### PR TITLE
Added dftx ICXCloseOrder

### PR DESCRIPTION
**Pull request by Jouzo**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:

Adds support to create ICXCloseOrder custom transactions from jellyfish side.

Which issue(s) does this PR fixes?:

Fixes part of #533

### CODE REVIEW

**Comment by jingyi2811** [comment]
There is a conflict. I think you created `dftx_icx_closeoffer` branch from this branch. You may need to apply code changes as per code review on  `dftx_icx_closeoffer` branch into this branch

**Review by surangap — COMMENTED** [review]
looks good. resolve conflicts.

File: packages/jellyfish-transaction/src/script/mapping.ts [hunk 1]
```diff
@@ -414,6 +424,7 @@ export const OP_CODES = {
       data: icxClaimDFCHTLC
     })
   },
+
```
Comment (surangap) [inline packages/jellyfish-transaction/src/script/mapping.ts:427]: extra line?

Comment (Jouzo) [reply packages/jellyfish-transaction/src/script/mapping.ts:427]: I reckon there was this extra line before we started merging the ICX DfTx

**Review by surangap — DISMISSED** [review]

**Review by fuxingloh — APPROVED** [review]
An additional test would be "attempting to close order that's not mine"

**Comment by Jouzo** [comment]
> An additional test would be "attempting to close order that's not mine"

Yes, let me add this one before merging it

**Review by jingyi2811 — APPROVED** [review]

**Review by fuxingloh — APPROVED** [review]

**Pull request merged**